lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -126,6 +126,20 @@ export class ResponseHandler {
                     throw ClientAuthError.createNonceMismatchError();
                 }
             }
+
+            // token max_age check
+            if (request.maxAge) {
+                const authTime = idTokenObj.claims.auth_time;
+
+                if (!authTime) {
+                    throw ClientAuthError.createAuthTimeNotFoundError();
+                }
+
+                const twoMinuteSkew = 120000; // two minutes in milliseconds
```

Comment:
nit: 2 * 60  * 1000 is easier to read, but I'm not sure if the TS transpiler or the JS interpreter will optimize this.

Reply:
I feel that the comment I left is adequate

Reply:
I can add a comma for readability